src/axom/sidre/tests/sidre_mfem_datacollection.cpp

```diff
@@ -774,6 +774,71 @@ TEST(sidre_datacollection, dc_par_reload_gf)
   EXPECT_TRUE(sdc_reader.verifyMeshBlueprint());
 }
 
+TEST(sidre_datacollection, dc_par_reload_gf_ordering)
+{
+  const std::string first_field_name = "test_field_1";
+  const std::string second_field_name = "test_field_2";
+
+  // 3D tet mesh
+  mfem::Mesh mesh(2, 2, 2, mfem::Element::TETRAHEDRON);
+  mfem::ParMesh parmesh(MPI_COMM_WORLD, mesh);
+
+  mfem::H1_FECollection fec(1, mesh.Dimension());
+  mfem::ParFiniteElementSpace first_parfes(&parmesh,
+                                           &fec,
+                                           1,
+                                           mfem::Ordering::byNODES);
+  mfem::ParFiniteElementSpace second_parfes(&parmesh,
+                                            &fec,
+                                            3,
+                                            mfem::Ordering::byVDIM);
```

Comment:
Based on the logic above, it would also be good to test a vector-valued grid function that is stored by nodes.
e.g.:
```cpp
  mfem::ParFiniteElementSpace third_parfes(&parmesh,
                                            &fec,
                                            3,
                                            mfem::Ordering::byNODES);
```